src/tooltip.tsx

```diff
@@ -105,6 +103,9 @@ type Props = {
     targetLabel: string;
     delay?: boolean;
     dataAttributes?: DataAttributes;
+    fullWidth?: boolean;
+    textAlign?: 'center' | 'left' | 'right' | 'justify';
+    changedPosition?: string;
```

Comment:
Just curious, what is this prop?

Reply:
fullWidth?: boolean; 
Necessary for aligning the Tooltip with the Thumb of the Slider component.

textAlign?: 'center' | 'left' | 'right' | 'justify';  
Removed. As requested, we leave only the options:
Left when the component is larger than minWidth
Centered when the component has the size of minWidth

changedPosition?: string;
This property makes the Tooltip change its position according to the space available for presentation without breaking